src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.ProgressAdapter.cs

```diff
@@ -36,7 +38,7 @@ public async Task OnReferenceFoundAsync(Document document, TextSpan span)
                 var documentSpan = await ClassifiedSpansAndHighlightSpanFactory.GetClassifiedDocumentSpanAsync(
                     document, span, _context.CancellationToken).ConfigureAwait(false);
                 await _context.OnReferenceFoundAsync(new SourceReferenceItem(
-                    _definition, documentSpan, SymbolUsageInfo.None)).ConfigureAwait(false);
+                    _definition, documentSpan, SymbolUsageInfo.None, additionalProperties: ImmutableArray<FindUsageProperty>.Empty)).ConfigureAwait(false);
```

Comment:
Nit: Should probably be called `FindUsagesProperty` instead of `FindUsageProperty` to match with `FindUsages` terminology used elsewhere.